### PR TITLE
update the basic feature tables for v6.4 

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11948
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by CbcWestwolf — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: basic-features.md [hunk 1]
```diff
@@ -184,9 +189,13 @@ This document lists the features supported in each TiDB version. Note that suppo
...
 | [Foreground Quota Limiter](/tikv-configuration-file.md#foreground-quota-limiter)  | Y | Y | Y| Experimental | Experimental | N | N | N | N | N | N |
+| [EBS volume snapshot backup and restore](https://docs.pingcap.com/tidb-in-kubernetes/v1.4/backup-to-aws-s3-by-snapshot)                        |    Y    |    N     |    N     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
+| [PITR](/br/point-in-time-recovery.md)   |    Y    |    Y     |    Y     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
 
 [^1]: TiDB incorrectly treats latin1 as a subset of utf8. See [TiDB #18955](https://github.com/pingcap/tidb/issues/18955) for more details.
 
 [^2]: See [Statement Reference](/sql-statements/sql-statement-select.md) for a full list of SQL statements supported.
 
-[^3]: For TiDB v4.0, the `LOAD DATA` transaction does not guarantee atomicity.
+[^3]: Starting from v6.4, TiDB supports [high-performance and globally monotonic `AUTO_INCREMENT` columns](/auto-increment#mysql-compatibility-mode)
```
Comment (Oreoxmt) [inline basic-features.md:199]:
```suggestion
[^3]: Starting from v6.4.0, TiDB supports [high-performance and globally monotonic `AUTO_INCREMENT` columns](/auto-increment#mysql-compatibility-mode)
```

File: basic-features.md [hunk 2]
```diff
@@ -10,156 +10,161 @@ This document lists the features supported in each TiDB version. Note that suppo
...
+| Multi-schema change: add columns                           | Y | Y | Y | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Change column type](/sql-statements/sql-statement-modify-column.md) | Y | Y | Y | Y | Y  | Y            |      Y       |      Y       |      Y       |      N       |      N       |
+| [Temporary tables](/temporary-tables.md)                    | Y | Y | Y | Y | Y   | Y            |      Y       |      N       |      N       |      N       |      N       |
+| [Concurrent DDL statements](/system-variables.md#tidb_enable_concurrent_ddl-new-in-v620) | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [Acceleration of `ADD INDEX` and `CREATE INDEX`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) | Experimental | Experimental | N | N | N | N | N | N | N | N | N |
+| [Metadata lock](/metadata-lock.md) | Experimental | Experimental | N | N | N | N | N | N | N | N | N |
+| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-to-timestamp.md) | Experimental | N | N | N | N | N | N | N | N | N | N |
```
Comment (Oreoxmt) [inline basic-features.md:102]:
```suggestion
| [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) | Experimental | N | N | N | N | N | N | N | N | N | N |
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: basic-features.md [hunk 3]
```diff
@@ -184,9 +189,13 @@ This document lists the features supported in each TiDB version. Note that suppo
...
 | [Foreground Quota Limiter](/tikv-configuration-file.md#foreground-quota-limiter)  | Y | Y | Y| Experimental | Experimental | N | N | N | N | N | N |
+| [EBS volume snapshot backup and restore](https://docs.pingcap.com/tidb-in-kubernetes/v1.4/backup-to-aws-s3-by-snapshot)                        |    Y    |    N     |    N     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
+| [PITR](/br/point-in-time-recovery.md)   |    Y    |    Y     |    Y     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
 
 [^1]: TiDB incorrectly treats latin1 as a subset of utf8. See [TiDB #18955](https://github.com/pingcap/tidb/issues/18955) for more details.
 
 [^2]: See [Statement Reference](/sql-statements/sql-statement-select.md) for a full list of SQL statements supported.
 
-[^3]: For TiDB v4.0, the `LOAD DATA` transaction does not guarantee atomicity.
+[^3]: Starting from v6.4.0, TiDB supports [high-performance and globally monotonic `AUTO_INCREMENT` columns](/auto-increment#mysql-compatibility-mode)
```
Comment (qiancai) [inline basic-features.md:199]:
```suggestion
[^3]: Starting from v6.4.0, TiDB supports [high-performance and globally monotonic `AUTO_INCREMENT` columns](/auto-increment.md#mysql-compatibility-mode)
```

**Pull request merged**